geocoding_v1.py

```diff
@@ -0,0 +1,133 @@
+""" This module provides functions to get the lat/lng for cities."""
+import json
+import time
+import urllib.error
+import urllib.parse
+import urllib.request
+from app import db, create_app
+from app.models import Cities
+from config import Config, DevelopmentConfig
+
+
+# Declaring Variables
+COUNTRY = "Netherlands"
+GOOGLE_API_KEY = Config.GOOGLE_API_KEY
+GEOLOCATE_BASE_URL = "https://maps.googleapis.com/maps/api/geocode/json"
+
+
+def get_coordinates(city):
+    """ Get the lat/lng for a city from the Google Maps Geolocate API.
+
+    Parameters:
+    city (string): The string for which the API will be queried.
+
+    Returns:
+    dict: {'lat': '...', 'lng': '...'}
+    """
+    params = urllib.parse.urlencode({
+        'address': f"{city},+{COUNTRY}",
+        'key': GOOGLE_API_KEY,
+    })
+    url = f"{GEOLOCATE_BASE_URL}?{params}"
+    print(url)
+    current_delay = 0.1
+    max_delay = 5
+
+    while True:
+        try:
+            # Get the API response:
+            response = urllib.request.urlopen(url)
+        except urllib.error.URLError:
+            pass
+        else:
+            result = json.load(response)
+            print(result)
+            if result['status'] == "OK":
+                return result['results'][0]['geometry']['location']
+            elif result['status'] == "ZERO_RESULTS":
+                return {'lat': '', 'lng': ''}
+            elif result['status'] != "UNKNOWN ERROR":
+                raise Exception(result['error_message'])
+
+        if current_delay > max_delay:
+            raise Exception("Too many retry attempts")
+
+        print('Waiting', current_delay, 'seconds before trying')
```

Comment:
I love the fact that this has retries 👍 nice!

I am just thinking - are you handling any Exceptions that were raised here?